framework/include/systems/NonlinearEigenSystem.h

```diff
@@ -114,7 +122,17 @@ class NonlinearEigenSystem : public NonlinearSystemBase
    * is the real and the imaginary part of
    * the eigenvalue, respectively.
    */
-  virtual const std::pair<Real, Real> getNthConvergedEigenvalue(dof_id_type n);
+  virtual const std::pair<Real, Real> getConvergedEigenvalue(dof_id_type n);
```

Comment:
If you're returning by value, is there a point to making the return value `const`? Also if returning by value, can you make this method `const`?

Reply:
Will do